tests/lint/files_exist.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+import os
+import yaml
+import nf_core.lint
+
+def test_missing_config(self):
```

Comment:
I've been using the convention of starting the function name with the name of the lint test. Then it's easy to run all pytests for a given lint check by using `-k <name>` with the pytest command.

```suggestion
def test_files_exist_missing_config(self):
```